src/test/java/scripts/CommonScriptTest.java

```diff
@@ -102,7 +102,7 @@ void testDetectVersionKey() throws IOException, InterruptedException {
     @Test
     void testDetectSource() throws IOException, InterruptedException {
         final Map<String, String> environment = createEnvironment(true);
-        environment.put(EnvironmentVariables.DETECT_SOURCE.name(), "https://sig-repo.synopsys.com/bds-integrations-release/com/synopsys/integration/synopsys-detect/5.1.0/synopsys-detect-5.1.0.jar");
+        environment.put(EnvironmentVariables.DETECT_SOURCE.name(), "https://repo.blackduck.com/bds-integrations-release/com/synopsys/integration/synopsys-detect/5.1.0/synopsys-detect-5.1.0.jar");
```

Comment:
Should we add a test for a post-sierra Detect jar?

Reply:
So the other tests which are not using any specifically assgined env variable are using the DETECT_LATEST_10 property configured using this line:

protected static final int DETECT_LATEST_VERSION = 10;

So, I believe we are already testing the post detect 10 jar as well.